lib/Default/SmrPlayer.class.inc

```diff
@@ -217,8 +217,35 @@ class SmrPlayer extends AbstractSmrPlayer {
 		else {
 			throw new PlayerNotFoundException('Invalid accountID: '.$accountIDOrResultArray . ' OR gameID:'.$gameID);
 		}
+	}
+
+	// Get array of players who are sharing info with this player
+	// Only returns players who are in the same alliance
```

Comment:
How do you get info about players who are sharing with you from another alliance? In general though I would expect this to be a parameter so you can choose to get just the same alliance or not

Reply:
In the current context, there is no circumstance under which you will share information between players who are not in the same alliance. Should the need arise in the future, such a parameter could be added, but I don't see any value gained in adding one now.